services/121-service/src/notifications/message-template/message-template.service.ts

```diff
@@ -21,7 +21,7 @@ export class MessageTemplateService {
   public async getMessageTemplatesByProgramId(
     programId: number,
     type?: string,
-    language?: string,
+    language?: LanguageEnum,
```

Comment:
Let's have a conversation about how this would have worked in practice. Of course then not knowing we would have to keep the branch without merging after finishing implementation.

Reply:
Yes indeed, this is part of a wider topic about feature branches. I will start a thread on teams about this and we can take it from there.